opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/EntityStixCoreRelationshipLineAll.jsx

```diff
@@ -113,48 +105,20 @@ class EntityStixCoreRelationshipLineAllComponent extends Component {
                 className={classes.bodyItem}
                 style={{ width: dataColumns.relationship_type.width }}
               >
-                <Chip
-                  variant="outlined"
-                  classes={{ root: classes.chipInList }}
-                  style={{ width: 120 }}
-                  color="primary"
-                  label={t(`relationship_${node.relationship_type}`)}
+                <ItemEntityType
+                  entityType={node.relationship_type}
+                  styles={{ width: 120 }}
```

Comment:
I think 120 is the default width anyway.

I think you are using either 120 or 140.

If this is really necessary, I would suggest to add a prop `size` to `ItemEntityType` with 'medium' giving 120 width and 'large' giving 140. if necessary, 'small' for 100 width.

My point is to avoid hardcoded values copy-pasted throughout the app, to keep UI consistent.